.terraform/main.tf

```diff
@@ -0,0 +1,73 @@
+terraform {
```

Comment:
This is all good but I would look to try creating a file on the latest version of Terraform. 👍🏼